tests/unit/test_service.py

```diff
@@ -35,8 +35,11 @@ def setUp(self):
         if hasattr(k8s_config, "ConfigurationObject"):
             p_mock_config = mock.patch.object(k8s_config,
                                               "ConfigurationObject")
-        else:
+        elif not hasattr(k8s_config.Configuration, "get_default_copy"):
```

Comment:
let's bump the min version of k8s client and remove these if..else blocks (even now our requirements.txt declares 5 as the minimum accepted version, but we still have these hacks for older versions)

Reply:
Agree. Which min version? 12.0.0 as OpenStack now proposes as upper-constraints.txt or 11.0.0 as the OpenStack latest releases?
If we want to remove all conditions, we do switch to 12.0.0 which breaks API.

Reply:
let's switch to 12.0.0